gpu_cmode.lua

```diff
@@ -174,7 +179,12 @@ function slurm_spank_task_exit(spank)
                    " on GPU(s): %s\n", default_cmode, device_ids)
     local cmd = nvs_path .. " -c " .. cmodes_index[default_cmode] ..
                             " -i " .. device_ids
-    local ret = tonumber(os.execute(cmd))
+    local ret = 0
+    if _VERSION == 'Lua 5.1' then
```

Comment:
Same as above (test for Lua <= 5.1)

Reply:
This has been updated to compare for less than or equal 5.1